app/src/main/java/ch/epfl/sweng/SDP/game/drawing/PaintView.java

```diff
@@ -138,6 +140,17 @@ public void setCircleRadius(int circleRadius) {
         this.circleRadius = circleRadius;
     }
 
+    public void setDrawWidth(int newWidth) {
+        drawWidth = newWidth;
+        for(Paint paint : colors) {
+            paint.setStrokeWidth(drawWidth);
+        }
+    }
+
+    public int getDrawWidth() {
```

Comment:
Here you can add VisibleForTesting

Reply:
No, this is used in DrawingActivity when a button is pressed.